virttest/qemu_vm.py

```diff
@@ -2924,14 +2924,8 @@ def create(self, name=None, params=None, root_dir=None,
             if params.get("paused_after_start_vm") != "yes":
                 # start guest
                 if self.monitor.verify_status("paused"):
-                    try:
+                    if not migration_mode:
                         self.monitor.cmd("cont")
-                    except qemu_monitor.QMPCmdError, e:
-                        if ((e.data['class'] == "MigrationExpected") and
-                                (migration_mode is not None)):
-                            logging.debug("Migration did not start yet...")
-                        else:
-                            raise e
```

Comment:
Well I'm afraid this might break the tests that use `not_wait_for_migration=True` (remote migrations and cpuflags test). I tried that cpuflags test, but somehow it does not work for me at all (even when filter-only 64bit archs).

Anyway I think in most cases this works well and the other tests would be broken even without this fix (as cont is not performed) so I believe it's fine to merge this and address the individual tests later.